Doc/tutorial/errors.rst

```diff
@@ -159,16 +201,13 @@ then re-raise the exception (allowing a caller to handle the exception as well):
        s = f.readline()
        i = int(s.strip())
    except OSError as err:
-       print("OS error: {0}".format(err))
+       print(f"OS error: {err}")
    except ValueError:
        print("Could not convert data to an integer.")
-   except BaseException as err:
+   except Exception as err:
        print(f"Unexpected {err=}, {type(err)=}")
```

Comment:
Personnally I find this formatting a little advanced for a tuto, but it was here before so it is what it is.